services/src/modules/arguments-injection/arguments-evaluation.ts

```diff
@@ -1,11 +1,12 @@
 import { VM } from 'vm2';
+import { argumentInjectionBuiltIns } from '../config';
```

Comment:
Don't really like the `BuiltIns` naming, I prefer `Helpers` (or `HelperFunctions` if you insist on being verbose)